fe/fe-core/src/main/java/com/starrocks/lake/compaction/CompactionScheduler.java

```diff
@@ -182,6 +185,9 @@ private void schedule() {
         int compactionLimit = compactionTaskLimit();
         int numRunningTasks = runningCompactions.values().stream().mapToInt(CompactionJob::getNumTabletCompactionTasks).sum();
         if (numRunningTasks >= compactionLimit) {
+            if (LOG.isDebugEnabled()) {
```

Comment:
specific reason to use `LOG.isDebugEnabled()`?

Reply:
Some sort of best practice to avoid unnecessary string splicing and other calculations, although not here, but to minimize the unnecessary mental burden, all the places where the debug logs are I've added if judgments